storage.go

```diff
@@ -45,8 +46,9 @@ func (s *Storage) copy(ctx context.Context, src string, dst string, opt pairStor
 	rs := s.getAbsPath(src)
 	rd := s.getAbsPath(dst)
 
+	srcPath := "/" + *s.properties.BucketName + "/" + url.QueryEscape(rs)
```

Comment:
We can use `service.StringValue`, which supported by `qingstor-sdk-go`, instead of get value from a pointer.